Uni_life_tracker/app/src/main/java/com/generic/ult/Settings_page.java

```diff
@@ -0,0 +1,44 @@
+package com.generic.ult;
+// import android.view.View;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+// import com.google.android.material.textfield.TextInputLayout;
+
+// import java.util.Objects;
+
+
+public class Settings_page extends AppCompatActivity {
+
+
+//    private TextInputLayout textInputName;
+//    private TextInputLayout textInputUni;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
Was wondering if the commented code here is needed

Reply:
I was thinking it would be useful for backend for setting up the buttons, I just couldn't get it to work at the moment so I commented it out

Reply:
I'll see if i can add that now